conf/messages

```diff
@@ -449,4 +449,10 @@ agent.enter-client-country.title=Enter your client’s country
 agent.enter-client-country.extraContent=This must match the country your client used in their Capital Gains Tax on UK property account contact address.
 
 postcode.error.noMatch=Postcode cannot be found
-countryCode.error.noMatch=Country cannot be found
\ No newline at end of file
+countryCode.error.noMatch=Country cannot be found
+
+agent.too-many-attempts.title=You’ll need to wait and try again
+agent.too-many-attempts.p1=You’ve entered incorrect details for your client too many times.
+agent.too-many-attempts.p2=You’ll need to wait 2 hours and try again.
```

Comment:
If this whole mechanism is designed to prevent phishing, then allowing this user to try again is bit pointless. Also when this user does exceed 10 tries, what happens then? Unless this is monitored and actioned upon, this mechanism is just giving a false sense of security IMO.